components/Storyblok/PageHeader/HeaderFullWidthImage.tsx

```diff
@@ -40,8 +41,8 @@ export const HeaderFullWidthImage = ({
         {fullWidthImage}
         {logoFilename && (
           <img
-            className="absolute max-sm:top-22 sm:max-md:top-58 md:bottom-120 lg:bottom-130 xl:bottom-140 2xl:bottom-150 left-1/2 -translate-x-1/2 z-20 max-h-[11.5rem] sm:max-h-[12.5rem] md:max-h-[13.7rem] lg:max-h-[17.6rem] xl:max-h-[22rem] 2xl:max-h-[26rem]"
-            src={logoFilename}
+            className="absolute max-sm:top-22 sm:max-md:top-58 md:bottom-120 lg:bottom-130 xl:bottom-140 2xl:bottom-150 left-1/2 -translate-x-1/2 z-20 max-h-[11.5rem] sm:max-h-[12.5rem] md:max-h-[13.7rem] lg:max-h-[17.6rem] xl:max-h-[22rem] 2xl:max-h-[26rem] max-w-300 sm:max-w-400 md:max-w-500"
+            src={getProcessedImage(logoFilename, `0x300`)}
```

Comment:
Processed the logo image instead of passing the raw storyblok image URL for the logo in `HeaderFullWidthImage`. See an example by inspecting the logo in the hero here (it now has the assets.stanford.edu domain:
https://deploy-preview-603--adapt-giving.netlify.app/get-involved/founders-circle